app/javascript/onboarding/__tests__/__snapshots__/Onboarding.test.jsx.snap

```diff
@@ -348,10 +348,8 @@ preact-render-spy (1 nodes)
             alt="DEV"
            />
         </figure>
-        <h1 class="introduction-title">firstname lastname— welcome to DEV!</h1>
-        <h2 class="introduction-subtitle">
-          DEV is where programmers share ideas and help each other grow.
-        </h2>
+        <h1 class="introduction-title">firstname lastname— welcome to Test!</h1>
+        <h2 class="introduction-subtitle">Wouldn't you like to knoww..</h2>
```

Comment:
Hm, how come the styles for “Wouldn’t you like to knoww..” change? 

It doesn’t look like you fiddled with the class so I wouldn’t expect a style change...